taker-frontend/src/components/Wallet.tsx

```diff
@@ -44,6 +111,65 @@ export default function Wallet(
                         <Timestamp timestamp={last_updated_at!} />
                     </Skeleton>
                 </HStack>
+
+                <Divider marginTop={2} marginBottom={2} />
+
+                <VStack padding={2}>
+                    <form onSubmit={runWithdraw}>
+                        <Heading as="h3" size="sm">Withdraw</Heading>
+                        <FormControl id="address">
+                            <FormLabel>Address</FormLabel>
+                            <Input
+                                onChange={(event) => onWithdrawAddressChange(event.target.value)}
+                                value={withdrawAddress}
+                                placeholder="Target address"
+                            >
+                            </Input>
+                        </FormControl>
+                        <HStack>
+                            <FormControl id="amount">
+                                <FormLabel>Amount</FormLabel>
+                                <NumberInput
+                                    min={0}
+                                    max={balance}
+                                    default={0}
+                                    onChange={(event) => onWithdrawAmountChange(event)}
```

Comment:
I think `event` is not true here, it is directly the amount? I don't see an code anywhere that extracts the number from this "event".

Reply:
On `NumberInput` this event is the value itself... don't ask me why

Reply:
It is not an `event` then is it? `value` would be a more appropriate name I think.

Reply:
yes, changed it already :)